bitsandbytes/functional.py

```diff
@@ -2508,3 +2536,49 @@ def vectorwise_mm_dequant(xq, S1, S2, dtype=torch.half, quant_type="vector"):
         return x.to(dtype)
     else:
         return None
+
+
+def enable_ipex_fusion(linear, x):
```

Comment:
This should probably not be a public API.

```suggestion
def _enable_ipex_fusion(linear: torch.nn.Module, x: torch.Tensor):
```